homework/index.js

```diff
@@ -30,13 +30,108 @@
     return elem;
   }
 
+  function update(val, dBase) {
+    const container = document.getElementById('container');
+    const leftDiv = createAndAppend('div', container, { class: 'left-div whiteframe' });
+    const table = createAndAppend('table', leftDiv);
+    const tbody = createAndAppend('tbody', table);
+    const rep = dBase[val];
+
+    for (let ix = 0; ix < 4; ix++) {
+      const row = createAndAppend('tr', tbody);
+      const tdFirst = createAndAppend('td', row, { class: 'label' });
+      const tdSec = createAndAppend('td', row);
+      const updateDate = new Date(rep.updated_at).toLocaleString();
```

Comment:
We don't need to do this for every loop iteration.

Reply:
Right.